subgraphs/compound-forks/protocols/iron-bank/src/constants.ts

```diff
@@ -1,11 +1,16 @@
 import { Address, dataSource, log, BigInt } from "@graphprotocol/graph-ts";
 import {
-  AVALANCHE_BLOCKS_PER_YEAR,
   ETHEREUM_BLOCKS_PER_YEAR,
   FANTOM_BLOCKS_PER_YEAR,
   Network,
+  SECONDS_PER_YEAR,
 } from "../../../src/constants";
 
+// See https://github.com/messari/subgraphs/issues/1746 for
```

Comment:
This is interesting, nice catch by the issue creator, and nice job jumping on it @tnkrxyz